docker-compose.yml

```diff
@@ -0,0 +1,20 @@
+version: "3.1"
+services:
+  auth2:
+    image: kbase/kb_auth2:${TRAVIS_BRANCH:-latest}
+    ports:
+      - "8080:8080"
```

Comment:
Do you really want to use such a common port?

Reply:
That turns out to be what the default for Jetty. This is not for production, the docker-compose file is just for testing in a mini environment

Reply:
ps.
   I just checked and I pretty much cribbed that jetty startup line from the prod entrypoint scripts, and the default port isn't reset so I'm pretty sure port 8080 is used in ci/appdev/prod for jetty